packages/frontend/src/hooks/useOrganizationGroups.ts

```diff
@@ -68,6 +69,41 @@ export const useGroupCreateMutation = () => {
     );
 };
 
+const updateGroupQuery = async (data: UpdateGroupWithMembers) =>
+    lightdashApi<GroupWithMembers>({
+        url: `/groups/${data.uuid}`,
+        method: 'PATCH',
+        body: JSON.stringify({
+            name: data.name,
+            members: data.members,
+        }),
+    });
+
+export const useGroupUpdateMutation = () => {
+    const queryClient = useQueryClient();
+    const { showToastSuccess, showToastError } = useToaster();
+    return useMutation<Group, ApiError, UpdateGroupWithMembers>(
+        (data) => updateGroupQuery(data),
+        {
+            mutationKey: ['update_group'],
+            onSuccess: async (_, updateGroup) => {
+                await queryClient.invalidateQueries(['update_group']);
```

Comment:
I think we can remove this line since there is no query with that key.

Reply:
Good catch